### PR TITLE
fix: enable Call for speaker

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/5315

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/api/speakers_calls.py [hunk 1]
```diff
@@ -43,14 +43,20 @@ def before_create_object(self, data, view_kwargs):
...
-            if (
-                speakers_call.starts_at > event.starts_at
-                or speakers_call.ends_at > event.starts_at
-            ):
-                raise ForbiddenError(
-                    {'source': ''},
-                    "Speakers call date can\'t be after the event start date",
-                )
+            if event.is_cfs_enabled:
+                if (
```
Comment (iamareebjamal) [inline app/api/speakers_calls.py:47]: Should be checked nevertheless

File: app/api/speakers_calls.py [hunk 2]
```diff
@@ -89,14 +95,20 @@ def before_patch(self, args, kwargs, data):
...
-                if (
-                    speakers_call.starts_at > event.starts_at
-                    or speakers_call.ends_at > event.starts_at
-                ):
-                    raise ForbiddenError(
-                        {'source': ''},
-                        "Speakers call date can\'t be after the event start date",
-                    )
+                if event.is_cfs_enabled:
+                    if (
```
Comment (iamareebjamal) [inline app/api/speakers_calls.py:99]: Should be checked nevertheless

Comment (maze-runnar) [reply app/api/speakers_calls.py:99]: means that event if cfs is not enabled, this check should be there ? 
```
           if (
                    speakers_call.starts_at > event.starts_at
                    or speakers_call.ends_at > event.starts_at
                ):
                    raise ForbiddenError(
                        {'source': ''},
                        "Speakers call date can\'t be after the event start date",
                    )
```

Comment (iamareebjamal) [reply app/api/speakers_calls.py]: Yes

**Pull request closed without merging**